index.js

```diff
@@ -0,0 +1,151 @@
+} from './modules/access-html-elements.js';
+
+// creating book object from class Book
+class features {
+  static addNewBook = () => {
+    // js to create a new book object
+    const newBookTitle = bookTitle.value;
+    const newBookAuthor = bookAuthor.value;
+    const book = new Book(newBookTitle, newBookAuthor);
+
+    // to set up and update local storage
+    if (localStorage.getItem('awesome-books') == null) {
+      localStorage.setItem('awesome-books', '[]');
+    }
+
+    // geting old data and merging with new data
+    const oldData = JSON.parse(localStorage.getItem('awesome-books'));
+    oldData.push(book);
+
+    // updating local storage data
+    localStorage.setItem('awesome-books', JSON.stringify(oldData));
+  };
+
+  static removeBook = (index) => {
+    const books = JSON.parse(localStorage.getItem('awesome-books'));
+    books.splice(index, 1);
+    // js to update local storage after removing a book
+    localStorage.setItem('awesome-books', JSON.stringify(books));
+  };
+
+  static displayBooks = () => {
+    container.innerHTML = '';
+    const books = JSON.parse(localStorage.getItem('awesome-books'));
+    for (let i = 0; i < books.length; i += 1) {
+      const div = document.createElement('div');
+      div.className = 'book-container';
+      if (i % 2 !== 0) {
+        div.className = 'white-color';
+      } else {
+        div.className = 'gray-color';
+      }
+
+      const textDiv = document.createElement('div');
+      textDiv.className = 'text-container';
+
+      const spanTitle = document.createElement('span');
+      spanTitle.className = 'book-title';
+      spanTitle.innerHTML = `"${books[i]?.title}"`;
+      textDiv.appendChild(spanTitle);
+
+      const by = document.createElement('span');
+      by.innerHTML = ' by ';
+      textDiv.appendChild(by);
+
+      const spanAuthor = document.createElement('span');
+      spanAuthor.className = 'author';
+      spanAuthor.innerHTML = books[i]?.author;
+      textDiv.appendChild(spanAuthor);
+
+      const removeButton = document.createElement('button');
+      removeButton.className = 'remove';
+      removeButton.innerHTML = 'Remove';
+      div.appendChild(removeButton);
+      removeButton.onclick = () => {
+        features.removeBook(i);
+        features.displayBooks();
+      };
+
+      div.appendChild(textDiv);
+      container.appendChild(div);
+    }
+  };
+}
+
+// js to add event to aadBook and prevent default
+const addNewBook = form.addEventListener('click', (event) => {
+  if (bookTitle.value == null || bookAuthor.value == null) {
+    event.preventDefault();
+  } else {
+    addBook.addEventListener('click', features.addNewBook);
+  }
+});
+
+// js to invoke displayBooks function
+const displayAllBooks = window.addEventListener(
+  'DOMContentLoaded',
+  features.displayBooks,
+);
+
+// js to display time
+const dateTime = DateTime.now();
+document.querySelector('.date-time').innerHTML = dateTime.toLocaleString(
+  DateTime.DATETIME_MED,
+);
```

Comment:
- [ ] [OPTIONAL] This is really awesome, chief, but I think it might look even more appealing if you set your time functionality in a `.setInterval` callback to ensure the time updates every second so that we don't have to refresh before getting actual time.
![image](https://github.com/EngruugbeJimmy/Awesome-App_New/assets/99715640/552eb711-1855-4cbe-8c7a-238a8c4741f0)